components/apimgt/org.wso2.carbon.apimgt.api/pom.xml

```diff
@@ -43,6 +43,11 @@
             <version>4.5.2</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.cxf</groupId>
+            <artifactId>cxf-rt-frontend-jaxrs</artifactId>
```

Comment:
why do we need this ?
cxf only resolves at webapps couldn't use  at bundle ime.

Reply:
removed